nrf_dm/README.rst

```diff
@@ -0,0 +1,161 @@
+.. _nrf_dm:
+
+Distance Measurement
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Distance Measurement is a library for measuring distance between Nordic devices (nRF52 Series).
+The library is available as hard-float builds for the nRF52 Series.
+
+Overview
+********
+
+This library provides the following key features:
+
+* Easy-to-use interface for distance measurement.
+* The distance is computed based on all information available to the device.
+* All the mathematics necessary to compute distance is performed inside the library.
+
+Background
```

Comment:
I am not familiar with the details of the ranging methods, but I've heard on some meetings that each method has its advantages in specific conditions: e.g. RTT might be more accurate for higher distances (I am not sure it's true, just an example). If any hint or characterization like this can be formulated, could you provide it here? It would help understand how to practically use this library.

Reply:
We agree that such information could be useful. At this point, we don't have all the information needed to provide such a statement with enough confidence, however. We would probably add more details about how to use the different modes as we approach the production release.

Reply:
Sounds good, thanks for clarifying.